### PR TITLE
[doc ] refactor of experiment content index

### PR DESCRIPTION
**Pull request by QuanluZhang**
- [x] the experiment doc skeleton
- [x] doc for local training service

### CODE REVIEW

File: docs/source/experiment/overview.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+NNI Experiment
+==============
+
+An NNI experiment is a unit of one tuning process. For example, it is one run of hyper-parameter tuning on a specific search space, it is one run of neural architecture search on a search space, or it is one run of automatic model compression on user specified goal on latency and accuracy. Usually, the tuning process requires many trials to explore feasible and potentially good-performing models. Thus, an important component of NNI experiment is **training service**, which is a unified interface to abstract diverse computation resources (e.g., local machine, remote servers, AKS). Users can easily run the tuning process on their prefered computation resource and platform. On the other hand, NNI experiment provides **WebUI** to visualize the tuning process to users.
```
Comment (JiahangXu) [inline docs/source/experiment/overview.rst:4]: Could you please explain what is exactly the relation between "experiment" and "training service"? Does the folder "/source/experiment/" only contain docs for training service? I'm a little confused.

File: docs/source/experiment/overview.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+During developing a DNN model, users need manage the tuning process, such as, creating an experiment, adjusting an experiment, kill or rerun a trial in an experiment, dumping experiment data for customized analysis. Also, users may create a new experiment for comparison, or concurrently for new model developing tasks. Thus, NNI provides the functionality of **experiment management**. Users can use :doc:`../reference/nnictl` to interact with experiments.
```
Comment (JiahangXu) [inline docs/source/experiment/overview.rst:6]: The first line should be "users need [to] manage"?

File: docs/source/experiment/overview.rst [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Before reading the following content, you are recommended to go through the quick start first.
```
Comment (ultmaster) [inline docs/source/experiment/overview.rst:8]: Link to quickstart?

File: docs/source/experiment/training_service.rst [hunk 4]
```diff
@@ -0,0 +1,18 @@
+Training Services
+=================
+
+NNI has supported many training services listed below. Users can go through each page to learning how to configure the corresponding training service. NNI has high extensibility by design, users can customize new training service for their special resource, platform or needs.
+
+..  toctree::
```
Comment (ultmaster) [inline docs/source/experiment/training_service.rst:6]: Suggest hiding the `toctree` with

```
.. toctree::
   :hidden:
```

And make another table like:

| Platform | Description (this header can be omitted) |
|------|-----|
| Local | Running trials on one single local machine. |
| Remote | .... |

File: docs/source/experiment/overview.rst (hunk 1 above)
Comment (QuanluZhang) [reply docs/source/experiment/overview.rst:4]: how about the updated version, I added a figure and one small paragraph. looking for your further comments

File: docs/source/experiment/overview.rst (hunk 2 above)
Comment (QuanluZhang) [reply docs/source/experiment/overview.rst]: fixed

File: docs/source/experiment/training_service.rst (hunk 4 above)
Comment (QuanluZhang) [reply docs/source/experiment/training_service.rst:6]: updated

File: docs/source/experiment/overview.rst (hunk 3 above)
Comment (QuanluZhang) [reply docs/source/experiment/overview.rst:14]: the quick start has not been updated, let's update this link later

**Review by JiahangXu — APPROVED** [review]

File: docs/source/experiment/local.rst [hunk 5]
```diff
@@ -0,0 +1,39 @@
+Local Training Service
+======================
+
+With local training service, the whole experiment (e.g., tuning algorithms, trials) runs on a single machine, i.e., user's dev machine. The generated trials run on this machine following `trialConcurrency` set in the configuration yaml file. If GPUs are used by trial, local training service will allocate required number of GPUs for each trial, like a resource scheduler.
```
Comment (ultmaster) [inline docs/source/experiment/local.rst:4]: ` ``trialConcurrency`` `

Comment (QuanluZhang) [reply docs/source/experiment/local.rst:4]: got it

**Review by ultmaster — APPROVED** [review]

**Pull request merged**